Detectors/TPC/workflow/src/CATrackerSpec.cxx

```diff
@@ -195,6 +199,17 @@ DataProcessorSpec getCATrackerSpec(ca::Config const& specconfig, std::vector<int
 
       config.configCalib.dEdxSplines = processAttributes->dEdxSplines.get();
 
+      auto& theMapper = o2::tpc::Mapper::instance();
+      std::string gainCorrectionFile{"GainMap_FromProductionData_round2.root"}; // TODO: set from flag
+      if (boost::filesystem::exists(gainCorrectionFile)) {
+        LOG(INFO) << "Loading tpc gain correction from file " << gainCorrectionFile;
+        const auto* gainMap = o2::tpc::utils::readCalPads(gainCorrectionFile, "GainMap")[0];
+        processAttributes->tpcCalibration.reset(new TPCCFCalibration{theMapper, *gainMap});
+      } else {
+        processAttributes->tpcCalibration.reset(new TPCCFCalibration{theMapper});
+      }
+      config.configCalib.tpcCalibration = processAttributes->tpcCalibration.get();
+
```

Comment:
The file should be set in `GPUSettingsO2` part in `GPUSettingsList.h` avaiable as `confParam` in `CATrackerSpec.cxx`, and it should be optional, as is already the case.